apps/lead/schema.py

```diff
@@ -395,7 +395,20 @@ def resolve_filtered_entries_count(root, info, **kwargs):
         return getattr(root, 'filtered_entry_count', None)
 
 
-class LeadDetailType(LeadType):
+class LeadTypeWithDuplicates(LeadType):
+    """Same as lead type but with duplicate_leads which is a recursive field"""
+    duplicate_leads = DjangoListField(LeadType)
+
+    class Meta:
+        model = Lead
+        only_fields = (
+            'id', 'title', 'is_assessment_lead', 'lead_group', 'assignee', 'published_on',
+            'text', 'url', 'attachment',
+            'client_id',
+        )
+
+
```

Comment:
Let's do this
- Add duplicates count in LeadType
- Add duplicates count 0 or >0 filter in LeadType Listing
- Add pagination query for DuplicateLeadsType with lead_id as an argument. In return use LeadType Listing